model.go

```diff
@@ -106,6 +106,9 @@ type CmdGroupModel struct {
 
 func (c *CmdGroupModel) FlattenedCommands() (out []*CmdModel) {
 	for _, cmd := range c.Commands {
+		if cmd.OptionalSubcommands {
+			out = append(out, cmd)
+		}
```

Comment:
If there is an optional subcommand then we add an additional flattened command string, otherwise we stay with the previous subcommand behavior which is to print the deepest possible chain of commands